### ISSUE TITLE
AVI mime type

### ISSUE THREAD

**Issue by moneytoo** [dburckh/ExoPlayer#4]
I understand that you define AVI with following mime type:

```java
public static final String VIDEO_AVI = BASE_TYPE_VIDEO + "/x-msvideo";
```

I also see this mime type in various lists (related to Android) but when I actually work with AVI files on Android, I usually have to use `video/avi` to:

* Filter items in MediaStore as indexed AVI files are stored with this mime type
* Filter files in native SAF file chooser as it also recognizes only `video/avi`
* 1st party apps like _Files by Google_ also use `video/avi`

I don't know the history of these mime types so these are just my observations.

**Comment by dburckh** [comment]
Yeah, I think that's wrong.  It does appear that the official mime type for avi is "video/x-msvideo", but a lot of times I'll see the extension in mime types.  I would use both.  :)

I think the way I have MP42 mapped is wrong, but I'm not sure what to map it to.  I did just see that my Note S20 has a video/mp43, so I'm going to change it to video/mp42.  I don't think it will work either way.  :)  I did get MP42 to play (poorly) when I mapped it to "video/mp43", but I don't think it's right.

Part of the issues is that container and the codec are both of type "video/", which confuses the matter.
